hazelcast-client/src/test/java/com/hazelcast/client/ClientServiceTest.java

```diff
@@ -26,14 +26,7 @@
 import com.hazelcast.client.test.TestHazelcastFactory;
 import com.hazelcast.config.Config;
 import com.hazelcast.config.ListenerConfig;
-import com.hazelcast.core.Client;
-import com.hazelcast.core.ClientListener;
-import com.hazelcast.core.ClientService;
-import com.hazelcast.core.EntryAdapter;
-import com.hazelcast.core.HazelcastInstance;
-import com.hazelcast.core.IMap;
-import com.hazelcast.core.LifecycleEvent;
-import com.hazelcast.core.LifecycleListener;
+import com.hazelcast.core.*;
```

Comment:
Can you check your idea settings for import with asterisk * ? We don't use * in our codebase. 
Settings -> Editor -> Code Style -> Java -> Imports (tab) 
Set "Class count to use import with" and "Names count to use static import with" to large numbers.